src/Microsoft.Identity.Client/HttpEvent.cs

```diff
@@ -30,35 +30,42 @@ namespace Microsoft.Identity.Client
 {
     internal class HttpEvent : EventBase
     {
+        public const string ConstHttpPath = "http_path";
+        public const string ConstUserAgent = "user_agent";
+        public const string ConstQueryParameters = "query_parameters";
+        public const string ConstApiVersion = "api_version";
+        public const string ConstResponseCode = "response_code";
+        public const string ConstOauthErrorCode = "oauth_error_code";
+
         public HttpEvent() : base(EventNamePrefix + "http_event") {}
 
         public string HttpPath
         {
-            set => this["http_path"] = value;
+            set => this[ConstHttpPath] = value;
```

Comment:
Now the tenant has been scrubbed by the last several commits. This PR is ready for review again.